classification/classifier.py

```diff
@@ -40,10 +42,10 @@ def classify(self):
             + self.contents.get_embedded_content_count() \
             + self.contents.get_quoted_content_count() >= self.min_pages
 
-    def is_page_stampifiable(self):
-        ''' returns the is_stampifiable flag'''
-        self.classify()
-        return self.is_stampifiable
+        if not self.is_stampifiable:
+            raise WebsiteNotStampifiableError(
+                message="Website cannot be stampified!",
+                failure_source="Summarizer")
```

Comment:
failure_source is "Summarizer"?

Reply:
Thanks for pointing this out. Will amend accordingly.